.travis.yml

```diff
@@ -1,21 +1,37 @@
 language: cpp
 
-os:
-  - linux
-  - osx
-
 # Set Xcode version to 10.1 (default is 9.4)
 osx_image: xcode10.1
 
 env:
   - VISP_INPUT_IMAGE_PATH=${TRAVIS_BUILD_DIR}
-  
-compiler:
-  - gcc
-  - clang
 
-sudo: required
-dist: trusty
+jobs:
+  include:
+    - os: linux
+      dist: xenial
+      arch: amd64
```

Comment:
Xenial 16.04 becomes default recently, but I'm wondering if we don't have to use bionic instead to test with 18.04.
What do you think ?

Reply:
There is also the `osx_image` that could be upgraded from 10.1 to 11.3
```
osx_image: xcode11.3
```

Reply:
Yes, I have switched to Bionic and one build with Xenial.
Same thing with Xcode.

Reply:
ok